docs/reference/eql/detect-threats-with-eql.asciidoc

```diff
@@ -0,0 +1,424 @@
+
+To get started, download and index the dataset:
+
+. Download the https://raw.githubusercontent.com/elastic/elasticsearch/{branch}/docs/src/test/resources/normalized-T1117-AtomicRed-regsvr32.json[`normalized-T1117-AtomicRed-regsvr32.json`] dataset.
+
+. Index the data into the `my-index-000001` index with the following
+<<docs-bulk,bulk API>> request:
++
+[source,sh]
+----
+curl -H "Content-Type: application/json" -XPOST "localhost:9200/my-index-000001/_bulk?pretty&refresh" --data-binary "@normalized-T1117-AtomicRed-regsvr32.json"
+----
+// NOTCONSOLE
+
+. Use the <<cat-indices,cat indices API>> to verify the dataset was indexed
+successfully.
++
+[source,console]
+----
+GET /_cat/indices/my-index-000001?v&h=health,status,index,docs.count
+----
+// TEST[setup:atomic_red_regsvr32]
++
+The API returns the following response. The `docs.count` value should show `150`
+documents were indexed.
++
+[source,txt]
+----
+health status index            docs.count
+yellow open   my-index-000001         150
+----
+// TESTRESPONSE[non_json]
+
+[discrete]
+[[eql-ex-get-a-count-of-regsvr32-events]]
+=== Get a count of `regsvr32.exe` events
+
+Since you're looking for `regsvr32` misuse, start by getting a count of events
+using `regsvr32.exe`.
+
+The following <<eql-search-api,EQL search API>> request uses an EQL query to
+retrieve a count of events with a `process.name` of `regsvr32.exe`. The query
+starts with the <<eql-syntax-match-any-event-category,`any where` keywords>>,
+meaning the query can match events of any <<eql-required-fields,event
+category>>.
+
+[source,console]
+----
+GET /my-index-000001/_eql/search?filter_path=-hits.events  <1>
+{
+  "query": """
+    any where process.name == "regsvr32.exe"        <2>
+  """,
+  "size": 150                                       <3>
+}
+----
+// TEST[setup:atomic_red_regsvr32]
+
+<1> Uses the `?filter_path=-hits.events` query parameter to exclude the
+`hits.events` property from the response. The `hits.events` property contains
+the document source for any matching events. This request is intended to
+retrieve a count of events only.
+<2> Uses an EQL query to match events with a `process.name` of `regsvr32.exe`.
+<3> Returns up to 150 events or sequences matching the EQL query.
+
+The request returns the following response, indicating that 143 events match the
+query.
+
+[source,console-result]
+----
+{
+  "is_partial": false,
+  "is_running": false,
+  "took": 60,
+  "timed_out": false,
+  "hits": {
+    "total": {
+      "value": 143,
+      "relation": "eq"
+    }
+  }
+}
+----
+// TESTRESPONSE[s/"took": 60/"took": $body.took/]
+
+[discrete]
+[[eql-ex-check-for-command-line-artifacts]]
+=== Check for command line artifacts
+
+Your previous query matched 143 events. Many of these events are likely to be
+false positives.
+
+To narrow down the results, check if the command line was used during any of
+these events. Append `and process.command_line > 0` to the previous EQL query.
```

Comment:
```suggestion
these events. Append `and process.command_line != null` to the previous EQL query.
```
I think it's also worth mentioning `event.category`, and by switching to `process where ...` you filter by category of `process`